src/utils/miscellaneous.ts

```diff
@@ -208,3 +208,5 @@ export const RequireContextMock: __WebpackModuleApi.RequireContext = Object.assi
 export function notEmpty<TValue>(value: TValue | null | undefined): value is TValue {
   return value !== null && value !== undefined
 }
+
+export const isStrNumTruthy = (value?: string | number): boolean => !!value && !!+value
```

Comment:
I had to scratch my head to understand it by name

what about sth like`isNonZeroNumber`?